vnext/Shared/Modules/WebSocketModule.cpp

```diff
@@ -396,6 +396,11 @@ shared_ptr<IWebSocketResource> WebSocketTurboModule::CreateResource(int64_t id,
 
 void WebSocketTurboModule::Initialize(msrn::ReactContext const &reactContext) noexcept {
   m_context = reactContext.Handle();
+  m_proxy = std::make_shared<WebSocketTurboModuleProxy>(m_resourceMap);
+
+  auto proxyPropId = ReactPropertyId<ReactNonAbiValue<weak_ptr<IWebSocketModuleProxy>>>{L"WebSocketModule.Proxy"};
```

Comment:
We should probably get a shared definition of this, so we dont have magic strings around the place -- and its clearer who is using this property.

Reply:
Can you please elaborate?
What is a "shared definition" in this context?

Reply:
At least make `L"WebSocketModule.Proxy"` a constant.  But ideally there would be a function or something that returned the `ReactPropertyId`, which could be used in all places using that property.